### PR TITLE
Hide conversation/Mute available on all chat rows

### PR DESCRIPTION
**Pull request by cjb**
@keybase/react-hackers CC @cecileboucheron

1. Add "Mute/Unmute all notifications" option to the chat gear-icon popup

2. Expand the desktop gear icon on hover to be available on adhoc conversations as well as teams

3. Expand the gear icon menu to be available on mobile when you long-press an inbox row

### CODE REVIEW

File: shared/chat/inbox/row/small-team/top-line.js [hunk 1]
```diff
@@ -37,16 +39,17 @@ class _SimpleTopLine extends React.Component<Props> {
     const boldStyle = this.props.showBold ? styles.bold : null
     return (
       <Kb.Box style={styles.container}>
-        {this.props.showGear && (
```
Comment (jzila) [inline shared/chat/inbox/row/small-team/top-line.js]: This line is necessary for the widget.

**Comment by cjb** [comment]
@jzila Thanks for the catch! Fixed that.

**Comment by cecileboucheron** [comment]
@cjb 
A few things: 
* username(s) should be semibold in the menu (like for small teams). And can we display the full name in BodySmall if it's just a 1-1 chat?
* In that tooltip, let's either display the "shh" in all cases, or never (currently it's missing in shh'd small teams)

**Comment by cjb** [comment]
@cecileboucheron Think I've done all of this now.

In order to get the shh overlay on the small team avatars, I reduced the mobile team avatar in the popup from 64px to 48px -- the TeamAvatar component we have can only take a shh overlay at 48px.

So now the avatar in the popup is always 48px across desktop and mobile.  Does that sound acceptable, at least for now?  (In the code, this is [shared/chat/avatars.js](https://github.com/keybase/client/blob/master/shared/chat/avatars.js).)

File: shared/chat/conversation/info-panel/menu/index.js [hunk 2]
```diff
@@ -65,29 +64,26 @@ const AdhocHeader = (props: AdhocHeaderProps) => (
...
       <Kb.Text type="BodySemibold">{props.teamname}</Kb.Text>
       <Kb.Text type="BodySmall">{`${props.memberCount} member${props.memberCount !== 1 ? 's' : ''}`}</Kb.Text>
     </Kb.Box2>
-  </>
+  </Kb.Box2>
 )
 
 class InfoPanelMenu extends React.Component<Props> {
-  componentDidDUpdate(prevProps: Props) {
+  componentDidUpdate(prevProps: Props) {
```
Comment (cjb) [inline shared/chat/conversation/info-panel/menu/index.js]: This typo was a bug in the existing code!  Checked that everything works with it fixed.

**Comment by cecileboucheron** [comment]
Sounds good! LGTM after my last comment (remove `shh` icon when it says Unmute)

**Pull request merged**